File: Odd or Even.cpp
//Program to find the given number is even or odd.
#include <iostream>
using namespace std;
int main()
{
    int x;
    cout << "Enter an integer: ";
    cin >> x; //Taking input from user
   
    //To Check if the number is even or odd 
    
    if ( x % 2 == 0){
        cout << x <<" "<< " is even.";
        }
    else{
        cout << x <<" "<< " is odd.";
        }
}

File: odd_or_even.py
# A function to check if a number is an odd or an even number
def check_number(number):
    if (number % 2) == 0:
        print(f"The number {number} is an even number.")
    else:
        print(f"The number {number} is an odd number.")

        
print("Check if your number is even or odd.")
num = int(input("Enter a number: "))
check_number(num)
